extensions/wikia/Chat2/js/controllers/ChatBanModalLogs.js

```diff
@@ -1,29 +1,29 @@
-var ChatBanLogController = function() {
-    $('.chat-change-ban').click(function(e) {
-        e.preventDefault();
-		var title = mw.html.escape($.msg('chat-ban-modal-change-ban-heading')),
-            userId = $(this).data('user-id'),
-			okCallback = function(expires, reason) {
-			 	$.post(wgScript + '?action=ajax&rs=ChatAjax&method=blockOrBanChat' ,{
+var ChatBanLogController = function () {
+	$('.chat-change-ban').click(function (event) {
+		var title = $.msg('chat-ban-modal-change-ban-heading'),
+			userId = $(this).data('user-id'),
+			okCallback = function (expires, reason) {
+				$.post(wgScript + '?action=ajax&rs=ChatAjax&method=blockOrBanChat', {
 						userToBanId: userId,
 						time: expires,
 						reason: reason,
 						mode: 'global',
 						token: mw.user.tokens.get('editToken')
 					},
-					$.proxy(function(data) {
+					function () {
```

Comment:
In addition I think if you use $.proxy with no second parameter
you bind 'this' to undefined